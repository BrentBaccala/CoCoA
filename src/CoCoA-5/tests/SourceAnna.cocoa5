RunningSourceAnna := True;
Source "anna.cocoa5";
RunningSourceAnna := False;
