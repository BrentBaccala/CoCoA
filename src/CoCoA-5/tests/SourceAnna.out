
# 
# 
-- INT, RAT --

-- LIST --

-- RING: PolyRing --

-- RINGELEM --

-- RING: RingTwinFloat --

-- MAT --

-- MAT --> MatrixView --

-- MAT --> MatrixArith --

-- MAT --> MatrixForOrdering --

-- IDEAL --

-- IDEAL: Hilbert --

-- IDEAL: toric --

-- IDEAL: monomial ideals --

-- RINGHOM --

-- QuotientRing --

-- MODULE --

-- MORE TESTS --

------ done ------

# 
# 